internal/build/build.go

```diff
@@ -49,6 +50,7 @@ var (
 	StagingAPIURL               = "https://api.stage.openshift.com"
 	DefaultClientID             = "rhoas-cli-prod"
 	DefaultOfflineTokenClientID = "cloud-services"
+	DefaultLoginTimeout         = 30 * time.Second
```

Comment:
This is way to low (in case when SSO session is not active it gives only 30 seconds to login)